### ISSUE TITLE
Relational noun coordination with outer possessive

### ISSUE THREAD

**Issue by nschneid** [UniversalConceptualCognitiveAnnotation/UCCA_English-EWT#5]
In https://github.com/UniversalConceptualCognitiveAnnotation/UCCA_English-EWT/commit/a786df3d0159a700d9ad588bed123e30b7c5c4c5:

"my wife and kids"
"my family or friends"
"my friends and family"

Shouldn't the possessive be repeated as a remote, as suggested at https://github.com/danielhers/ucca/issues/62? I.e. "my wife and (my) kids".

Consider that the possessive could be used with a coordinated relational noun and nonrelational noun:

my kids and dog
[my_A kids_S+A]_C and_N [(my)_S+A dog_A]_C

**Comment by dotdv** [comment]
All these three cases were marked with CMR (you can't see it in the github file but it appears in the annotation). I see in  [danielhers/ucca#62](url) that you want to cancel this option? OK, so should I fix these three?
For the record, the task ids are:
test 2843 my family or friends
dev 4093 my wife and kids
dev 4054 my friends and family

**Comment by omriabnd** [comment]
Yes. I think we should avoid CMR in these cases, since it's a bit confusing.

**Comment by dotdv** [comment]
done
